tesseract/integration-test/src/lib.rs

```diff
@@ -476,6 +396,8 @@ async fn run_integration_tests() -> Result<(), anyhow::Error> {
 		set_host_params(chain_b_sub_client.clone()).await?;
 	}
 
+	tokio::time::sleep(Duration::from_secs(6)).await;
```

Comment:
Use subscribe block headers to know when the parachain has started block production instead of sleeping.

You should do that before calling `set_host_params`